### PR TITLE
Switch to code generator sample files

### PR DESCRIPTION
**Pull request by JamesPaden**
These files have been programmatically generated and tested. Using them will work to standardize our code samples across all agents and documentation.

https://3.basecamp.com/3093825/buckets/1202046/todos/4410699987

### CODE REVIEW

File: examples/Async.java [hunk 1]
```diff
@@ -5,58 +5,60 @@
...
 //
 // You can run this example with: ./script/run_java_file examples/Async.java
 
+import com.docraptor.*;
 import java.io.*;
 import java.net.*;
-import com.docraptor.*;
+import java.nio.file.*;
+
+public class Sync {
```
Comment (jason-o-matic) [inline examples/Async.java:18]: Seems like this class name should roughly match the filename. Perhaps `AsyncTest` in order to indicate it's a test as well? I'm not super sure of the conventions for test class names in java.

**Review by jason-o-matic — CHANGES_REQUESTED** [review]
Mostly good, the test class names are just confusing.

File: examples/Async.java (hunk 1 above)
Comment (JamesPaden) [reply examples/Async.java:18]: What if it was just `DocRaptor`? And it's not a test, it is just tested to ensure it works before publishing.

Comment (jason-o-matic) [reply examples/Async.java:18]: Ah, yeah, that makes sense. My brain had conflated the tests and the examples in these repos. Maybe we can DRY up the tests at some point as well, but that can be later. 

Yeah, `DocRaptor` seems okay, maybe `DocRaptorExample` to differentiate it from the official DocRaptor library classes?

Comment (JamesPaden) [reply examples/Async.java:18]: `DocRaptorExample` sounds good. Thanks!

Comment (JamesPaden) [reply examples/Async.java]: Fixed

**Review by jason-o-matic — APPROVED** [review]

**Pull request merged**